doc/API.md

```diff
@@ -608,6 +613,120 @@ In axe-core v1 the main method for axe was `axe.a11yCheck()`. This method was re
 - .a11yCheck requires a context object, and so will not fall back to the document root.
 - .a11yCheck does not return a Promise.
 
+### Virtual DOM Utilities
+
+#### API Name: axe.utils.getFlattenedTree
+
+##### Description
+
+Recursvely return an array containing the virtual DOM tree for the node specified, excluding comment nodes and shadow DOM nodes `&lt;content> and `&lt;slot>`. This method will return a composed tree containing both light and shadow DOM, if applicable.
+
+##### Synopsis
+
+```javascript
+var element = document.body;
+axe.utils.getFlattenedTree(element, shadowId)
+```
+
+##### Parameters
+ - `node` – node. The current HTML node for which you want a flattened DOM tree.
+ - `shadowId` – string(optional). ID of the shadow DOM that is the closest shadow ancestor of the node
```

Comment:
I'm not quite following. Can't we set it as `undefined` if it doesn't get passed in? This may also be moot if we drop documentation of this function.